Wireframe/index.html

```diff
@@ -1,13 +1,67 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="styles.css" />
   </head>
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
-</html
\ No newline at end of file
+
+    <div class="main">
+      <header>
+        <h1>Sustainable Life & Wellness</h1>
+        <h5>"Inspiring Conscious Choices for a Greener Life"</h5>
+      </header>
+      <div class="chaild1">
+        <h2>The Benefits of Daily Meditation</h2>
+        <article>
+          Meditation is a powerful tool for reducing stress and improving mental
+          clarity. Just 10 minutes a day can lead to better focus, emotional
+          health, and overall well-being. Whether through mindfulness or guided
+          sessions, regular practice helps to calm the mind and reduce anxiety.
+        </article>
+        <a href="https://www.healthline.com/nutrition/12-benefits-of-meditation"
+          >Read More</a
+        >
+      </div>
+      <div class="article">
+        <div class="chaild2">
+          <h3>Sustainable Fashion Trends</h3>
+          <article>
+            Sustainable fashion is more than a trend; it's a movement towards
+            ethical production and eco-friendly materials. Brands are focusing
+            on reducing waste, using organic fabrics, and ensuring fair labor
+            practices. Investing in sustainable fashion can help reduce your
+            carbon footprint and support ethical industries.
+          </article>
+          <a
+            href="https://clothesaid.co.uk/13-key-sustainable-fashion-trends-in-2024-confronting-the-urgent-challenges-of-a-greener-future/"
+            >Read More</a
+          >
+        </div>
+        <div class="chaild3">
+          <h4>The Rise of Electric Vehicles</h4>
+          <article>
+            Electric vehicles (EVs) are revolutionizing transportation, offering
+            eco-friendly alternatives to gas-powered cars. With advances in
+            battery technology and an increasing number of charging stations,
+            EVs are becoming more accessible and affordable. Many countries are
+            pushing for a greener future with EV incentives.
+          </article>
+          <a
+            href="https://www.iea.org/energy-system/transport/electric-vehicles"
+            >Readmore</a
+          >
+        </div>
+      </div>
+      <div class="chaild4">
+        <footer>
+          © 2024 Nastaran. Inspiring change, one mindful step at a time
+        </footer>
+      </div>
```

Comment:
In this case there was no need to give footer a div and class="chaild4". Simply a <footer> could have worked, and in css just referenced footer{}